rj_gameplay/rj_gameplay/tactic/pass_or_shoot.py

```diff
@@ -0,0 +1,134 @@
+from dataclasses import dataclass
+from typing import List, Optional, Callable
+import stp.rc as rc
+import stp.tactic as tactic
```

Comment:
Why'd this file get marked as checked in? Did you check it out from the comp branch?

Reply:
^ as figured out in last subteam meeting, merging into `comp-2021-merge` branch should hopefully fix this